src/remediations/fifi.js

```diff
@@ -247,19 +247,22 @@ function defineRHCEnabledExecutor (satellites, smart_management, rhc_enabled) {
             satellites.push({id: null, systems: partitionedSystems[0], type: 'RHC', rhcStatus: (rhc_enabled) ? CONNECTED : DISABLED});
         }
 
-        satlessExecutor.systems = partitionedSystems[1];
+        satlessExecutor.systems = _.filter(partitionedSystems[1], system => _.isUndefined(system.rhc_client) && !system.marketplace);
```

Comment:
Is the satless executor redundant now that we have `no_rhc` and `no_smart_management` states? The 2 of themshould capture all subtypes of "direct connection" hosts, or?

Reply:
@jharting so I "think" that the satless executor state could still be a valid state for the pre-flight check since it is technically possible to have a system that both.
1. does not have a rhc_client_id and has not been registered with cloud connector.
2. is not a marketplace system and the Customer does not have the smart_management entitlement

This would definitely be a rare occurrence though.

Reply:
So if a system satisfies both `no_rhc` and `no_smart_management` at the same time then it does not end up in either of those two buckets but instead goes to a third `satless executor` one?

Also, from reading the code it looks like if a customer has smart management, the host is not a marketplace one and does not have `rhc_client_id` then the host will end up in `satless executor` also (instead of no_rhc)

Reply:
Yes, I believe that's right, should a system satisfy both of those requirements we have the "no executor" bucket to put it in.

Also Ah!  yes you're right.  I'm going to remedy that, ty.

Reply:
> Yes, I believe that's right, should a system satisfy both of those requirements we have the "no executor" bucket to put it in.

Does `no_executor` that map to any of the states in https://marvelapp.com/prototype/6cf654e/screen/77431133 ?

Reply:
I believe it's currently mapping to it's old state, i.e. Cannot remediate because of direct connect.  I'm chatting with Robin rn to make it something else.